packages/@headlessui-react/src/components/combobox/combobox-machine-glue.tsx

```diff
@@ -16,8 +16,9 @@ export function useComboboxMachineContext<T>(component: string) {
 export function useComboboxMachine({
   virtual = null,
   __demoMode = false,
+  id = globalThis.crypto.randomUUID() as string,
```

Comment:
Maybe this is fine but function only works on https & localhost. Could cause things to break for virtual hosts that don't have HTTPS enabled in development.

Maybe replacing it with Math.random() is _good enough_?

Reply:
Or https://github.com/tailwindlabs/headlessui/pull/3726#discussion_r2085199887 haha

Reply:
Yeah if we can just remove it then we should 👍